virtual-desktop/src/app/plugin-manager/shared/plugin-loader.ts

```diff
@@ -47,9 +47,8 @@ export class PluginLoader {
     });
   }
 
-  loadPlugin(pluginDefinition: MVDHosting.DesktopPluginDefinition): Promise<CompiledPlugin> {
+  loadPlugin(pluginDefinition: MVDHosting.DesktopPluginDefinition, windowId: MVDHosting.ViewportId): Promise<CompiledPlugin> {
```

Comment:
windowId != viewportId because a window can contain multiple viewports, although this isn't utilized much in the existing apps.

Reply:
Thanks for pointing that out.